plugin/storage/cassandra/spanstore/writer_test.go

```diff
@@ -314,6 +314,40 @@ func TestStorageMode_IndexOnly(t *testing.T) {
 	}, StoreIndexesOnly())
 }
 
+type indexFilter struct{}
```

Comment:
This filter will filter out everything, right? Would it be more appropriate to name it `noNameFilter{}`?